packages/rancher-istio-1.8/charts/values.yaml

```diff
@@ -82,7 +82,10 @@ kiali:
       custom_metrics_url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
       url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
     tracing:
-      in_cluster_url: "http://tracing.istio-system.svc:16686/jaeger"
+      auth:
+        insecure_skip_verify: true
+      in_cluster_url: "https://173.255.241.30/k8s/clusters/c-zbpdp/api/v1/namespaces/istio-system/services/http:tracing:16686/proxy/jaeger"
+      url: "https://173.255.241.30/k8s/clusters/c-zbpdp/api/v1/namespaces/istio-system/services/http:tracing:16686/proxy/jaeger"
```

Comment:
Are these IP's ok to be in the chart?

Reply:
I did mean to change this to a generic url - I have updated that with my most recent commit